Testing/Unit/PowerShell/CreateReport/New-Report.Tests.ps1

```diff
@@ -0,0 +1,111 @@
+BeforeAll {
+    Import-Module (Join-Path -Path $PSScriptRoot -ChildPath '../../../../PowerShell/ScubaGear/Modules/CreateReport')
+    New-Item -Path (Join-Path -Path $PSScriptRoot -ChildPath "./CreateReportStubs") -Name "CreateReportUnitFolder" -ErrorAction SilentlyContinue -ItemType Directory | Out-Null
+    New-Item -Path (Join-Path -Path $PSScriptRoot -ChildPath "./CreateReportStubs/CreateReportUnitFolder") -Name "IndividualReports" -ErrorAction SilentlyContinue -ItemType Directory | Out-Null
+}
+
+Describe -Tag CreateReport -Name 'New-Report' {
```

Comment:
In general, the tests do not fully verify the logic of the unit under test.  Consider adding additional checks to verify correct paths are transfered based on appropriate input.

Reply:
more detailed unit testing will be coming. Will make an issue for it as well.